src/renderer/widgets/WalletDetails/model/vault-details-model.ts

```diff
@@ -0,0 +1,38 @@
+import { createStore, createEvent, forward, createEffect, sample } from 'effector';
+
+import type { ShardAccount, Chain, ChainId } from '@shared/core';
+import { chainsService } from '@entities/network';
+
+const shardsSelected = createEvent<ShardAccount[]>();
+const shardsCleared = createEvent();
+
+const chainSetFx = createEffect((chainId: ChainId): Chain | undefined => {
```

Comment:
I think we should discuss order events/store/effects, because I thought that we have events -> stores -> effects

Reply:
Good point, some stores might depend on events (`reset` events for example)